README.md

```diff
@@ -1,148 +1,165 @@
+        <version>1.8</version>
+        <executions>
+          <execution>
+            <id>ktlint</id>
+            <phase>verify</phase>
+            <configuration>
+              <target name="ktlint">
+                <java taskname="ktlint" dir="${basedir}" fork="true" failonerror="true"
+                  classpathref="maven.plugin.classpath" classname="com.pinterest.ktlint.Main">
+                  <arg value="src/**/*.kt"/>
+                </java>
+              </target>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>com.pinterest</groupId>
+            <artifactId>ktlint</artifactId>
+            <version>0.37.2</version>
+          </dependency>
+          <dependency>
+            <groupId>org.cqfn.diktat</groupId>
+            <artifactId>diktat-rules</artifactId>
+            <version>0.0.1-SNAPSHOT</version> <!-- replace it with our current version -->
+          </dependency>
+        </dependencies>
+      </plugin>
+    </plugins>
+  </build>
+</project>
 ```
 
-in case you want to add autofixer with diktat ruleset just extend the snippet above with: `<arg value="-F"/>`
+In case you want to add autofixer with diKTat ruleset just extend
+the snippet above with `<arg value="-F"/>`.
+
+## Customizations via `rules-config.json`
+
+In KTlint, rules can be configured via `.editorconfig`, but
+this does not give a chance to customize or enable/disable
+each and every rule independently.
+That is why we have supported `rules-config.json` that can be easily
+changed and help in customization of your own rule set.
+It has simple fields:
+`name` — name of the rule,
+`enabled` (true/false) — to enable or disable that rule, and
+`configuration` — a simple map of some extra unique configurations for the rule.
+For example:
 
-### How to: make deep customization of diKTat (rules-config.json)
-In ktlint rules can be configured via .editorconfig, but this does not give a chance to customize or enable/disable each and every rule independently.
-That is why we have supported `rules-config.json` that can be easily changed and help in customization of your own rule set. 
-It has simple fields: "name" - name of the rule, "enabled" (true/false) to enable or disable that rule and "configuration" - a simple map of some extra unique configurations for the rule, for example: 
 ```json
- "configuration": {
-      "isCopyrightMandatory": true,
-      "copyrightText": "Copyright (c) Huawei Technologies Co., Ltd. 2012-2020. All rights reserved."
-    }
+"configuration": {
+  "isCopyrightMandatory": true,
+  "copyrightText": "Copyright (c) Jeff Lebowski, 2012-2020. All rights reserved."
+}
 ```
 
-To see all list of available configurations go to our wiki: [Available inspections](info/available-rules.md)
+## How to contribute?
 
-## Development
```

Comment:
You have removed all these titles, do you think that everything became readable?

Reply:
@akuleshov7 yes, they are not needed in such a short document. they just distract the reader